azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -873,6 +913,31 @@ def start_tracking_extension_services_cgroups(self, services_list):
                     if service_name is not None:
                         self.start_tracking_unit_cgroups(service_name)
 
+        @staticmethod
+        def get_extension_services_list():
```

Comment:
retrieve the manifest files of installed extensions to reset quotas when we disable cgroups for all.